www/templates/jfoundation/css/custom.css

```diff
@@ -56,6 +56,11 @@ body.site.fluid {
     border-radius: 0;
 }
 
+/* addition of background color change of navbar items on hover  */
+.navigation .nav li > a{
```

Comment:
This selector does not target the nav items on hover. It also needs to include the focus state.